src/style.css

```diff
@@ -0,0 +1,272 @@
+.calendar {
+  box-sizing: border-box;
+  padding: 10px;
+  width: 732px;
+  min-height: 500px;
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: 1px;
+}
+
+.day {
```

Comment:
The class `.day` is used here, but in the HTML file, the class `calendar__day` is used. Ensure that the class names match between the HTML and CSS files to apply the styles correctly.